templates/event/base.html

```diff
@@ -3,6 +3,14 @@
 <!DOCTYPE HTML>
 <html>
 <head>
+  <!-- Google tag (gtag.js) -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-JGJW1VEY0D"></script>
```

Comment:
This gets loaded regardless of the cookie consent choice, so what functionality is listening out for that consent?

Reply:
Thanks, let me add that functionality.

Reply:
@marksweb, I followed the guidelines here https://developers.google.com/analytics/devguides/collection/gtagjs/user-opt-out to disable the google tag on cookie reject.